docs/modules/integrations/pages/azure-devops-exporter.adoc

```diff
@@ -47,6 +47,25 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 |`AUTOMATED` or `MANUAL`
 |The scenario part to export into `Steps` section of `Test Case` work item, the remaining scenario part is exported into `Summary` section.
 
+|`azure-devops-exporter.test-run.name`
+|
+|string
+a|The test run name.
+
+Setting of this property leads to creation of a new test run with the specified name.
+
+[IMPORTANT]
+====
+* All Azure DevOps Test Cases specified by the `testCaseId` meta tags on VIVIDUS scenarios within current VIVIDUS test run should belong the the Azure DevOps Test Plan pointed by the `azure-devops-exporter.test-run.plan-id` property.
+* Its not allowed to mix `manual` (contains only comments) and `automated` (contains at least one actionable step) test cases within one Azure DevOps Test Run.
```

Comment:
the definition of manual and automated scenarios should be moved out from Xray to a some common section, then this section should be referenced from here and Xray guides